### ISSUE TITLE
Add RSS format for Friendica published feeds

### ISSUE THREAD

**Issue by tiotrom** [friendica/friendica#9317]
I can grab an Atom feed, but can I grab an RSS feed? If not, could it be implemented?

**Comment by MrPetovan** [comment]
Both are supposed to be supported. I haven't encountered an Atom/RSS feed that Friendica couldn't parse. Do you have a specific instance of an RSS feed not working?

**Comment by AlfredSK** [comment]
Yes, you can.

And while it is okay to ask support questions here on the issue tracker I want to give a polite hint that we have a support forum within Friendica itself. 
`!helpers@forum.friendi.ca`

**Comment by tiotrom** [comment]
I meant that I can grab the Atom feed from here https://social.isurf.ca/feed/trom/posts but I am looking for an RSS feed. So I am talking about an RSS feed for Friendica posts. Sorry I wasn't very specific at all.

@AlfredSK Thank you sir :). I will try to use that from now on. Let's keep my latest submissions here and if I need more help I'll ask there.

**Comment by MrPetovan** [comment]
No, we don't publish both feeds for Friendica posts, just the Atom one. What's your use case? Why is it restricted to RSS?

**Comment by tiotrom** [comment]
Ah...we use an RSS Reader for our website to pull our posts from Friedrica as "live-updates". Basically we are a handful of people working on our project at tromsite.com and now we are moving to Friendica and we post updates there about the project. But we would like to display our updates on our website too, like we've always did, and since there is no other way to integrate Friedrica with our Wordpress website for thus job, I thought to simply pull the RSS from Friedrica and create a stream with that. But the tool we use to grab the Feed only works with RSS and not Atom...so one way I did it is to put it through FeedBurner (google) to convert Atom to RSS....

But I would like not to use Google for this. So I thought you may provide both Atom and RSS for Friedica. Why not provide an RSS?

**Comment by MrPetovan** [comment]
The simple and sad answer is because it is about 50% more work. We have to support subscribing to both Atom and RSS feeds because both exist out there, but we don't have to publish both for us to be able to tick the post feed feature box.

**Comment by tiotrom** [comment]
I see. RSS Readers should be able to read most of these. Unfortunately the one we use is not...do you plan to support RSS too in the future?

**Comment by annando** [comment]
That's currently not planned. But you could open a feature request. Possibly someone wants to do it.

**Comment by tiotrom** [comment]
Ok. I posted in the feature request tho, so is it ok to leave it here?

**Comment by hoergen** [comment]
Since the feature request is opened, could this issue be closed?

**Comment by MrPetovan** [comment]
This issue is the feature request. 🙂

**Comment by hoergen** [comment]
Oh pardon :)  Too many similar issues and tabs. Thanks for clarifying :+1:
